about.md

```diff
@@ -6,294 +6,48 @@ title: About MDAnalysis
+## Funding & Sponsors
 
-MDAnalysis and the algorithms implemented in the library and the subprojects are
-scientific software that are described in [academic
-publications]({{site.baseurl}}/pages/citations/).
+MDAnalysis is supported by grants, sponsorships, and community contributions. Organizations such as the **Chan Zuckerberg Initiative, Google, NumFOCUS, and the National Science Foundation** have played key roles in funding our development.
 
-MDAnalysis is used in a variety of [other molecular dynamics tools]({{site.baseurl}}/pages/mdakits/).
+For more information on our funding sources and how you can support MDAnalysis, visit our [Funding]({{ site.baseurl }}/pages/funding/) page.
 
+## Open Source & Availability
 
-## Open source
+MDAnalysis is released under the [GNU Lesser General Public License, version 2.1 or later (LGPLv2.1+)][]. This ensures that our tools remain accessible, transparent, and reproducible for molecular dynamics research. Some supporting libraries use the MIT or BSD-3 licenses.
 
-All our [code]({{site.baseurl}}/#availability) and our [teaching
-materials]({{site.baseurl}}/pages/learning_MDAnalysis/) are available
-under open source licenses from repositories at
-[https://github.com/MDAnalysis/](https://github.com/MDAnalysis/). The
-MDAnalysis library itself is published under the [GNU General Public
-License, version 2](https://www.gnu.org/licenses/gpl-2.0.html); other
-supporting libraries are published under the MIT or the BSD-3 clause
-licence. 
+MDAnalysis is: 
 
-Installable packages are available through the popular ``pip`` and
-``conda`` package managers as well as some Linux distributions.
+- **Easy to install** &mdash; Available via `pip` and `conda/mamba`.
+- **Openly developed** &mdash; The [source code]({{ site.github.repo }}) is hosted on GitHub.
+- **Regularly updated** &mdash; Official releases are distributed via [PyPI][].
 
+For full installation instructions and setup guides, visit our [Getting Started]({{ site.baseurl }}/pages/getting_started/) page.
 
-## Governance
+## Share Your Feedback
 
-Project leadership is provided by a subset of contributors, the
-*MDAnalysis Core Developers*
-([@MDAnalysis/coredevs](https://github.com/orgs/MDAnalysis/teams/coredevs))
-who have produced substantial contributions over extended lengths of
-time and who remain active in reviewing issues and discussions on
-GitHub Discussions and our Discord server. 
+MDAnalysis welcomes feedback for improvement from its users and community. Share your thoughts on MDAnalysis, the community, or events by filling out [this feedback form][].
 
-### MDAnalysis Core Developers
+[GNU Lesser General Public License, version 2.1 or later (LGPLv2.1+)]: https://www.gnu.org/licenses/lgpl-2.1.html
+[PyPI]: https://pypi.org/project/MDAnalysis/
+[this feedback form]: https://forms.gle/n8GLe2QsL2hW2QiDA
+[Core Developers]: https://github.com/orgs/MDAnalysis/teams/coredevs/members
```

Comment:
This is actually not a good link. The only real list of current CoreDevs is on the web page, so link to that list.